include/param_defs.h

```diff
@@ -3,7 +3,9 @@
 
 #include <vector>
 
-#include "parameters/parameter.h"
+#include "lsst/modelfit/parameters.h"
+
+namespace parameters = lsst::modelfit::parameters;
```

Comment:
This will cause any source file that (directly or indirectly) includes this header to define `parameters`. Potential namespace collision if it's also using some other parameters?